exo-go/src/types/tf_string.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import "strings"
+
+// TFString is an abstract of a string representing a .tfvars file
+type TFString string
+
+// ToSecrets converts .tfvars string to .tfvars map:
+// a="b"
+// c="d" ->
+// {a:b, c:d}
+func (str TFString) ToSecrets() Secrets {
+	secretsMap := Secrets{}
+	secretPairs := strings.Split(string(str), "\n")
+	if len(secretPairs) > 1 {
+		for _, secret := range secretPairs {
+			s := strings.Split(secret, "=")
+			secretsMap[s[0]] = strings.Trim(s[1], "\"")
+		}
+	}
+	return secretsMap
+}
```

Comment:
Thinking about this more I don't think we need the TFString type. We could get by with just the Secrets struct and add a NewSecrets methods which takes in a string and parses it into a map (same logic as this function)